test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Date.Tests.ps1

```diff
@@ -172,6 +172,11 @@ Describe "Get-Date" -Tags "CI" {
         $timeDifference.Milliseconds | Should -BeLessThan 1
         $timeDifference.Ticks        | Should -BeLessThan 10000
     }
+
+    It "-FromUnixTime works" {
+        Get-Date -Date 1577836800 -FromUnixTime | Should -Be (Get-Date -Date 637134336000000000)
+        Get-Date -Date 0 -FromUnixTime | Should -Be (Get-Date -Date 621355968000000000)
```

Comment:
Please add a comment about these used constants.
Also right part should return UTC, not Local.

Reply:
Good catch! I've added comments to those tests 😅

From what I can find, the `Get-Date -Date` cmdlet by itself will return a `DateTime` with the Kind set to `Unspecified`. Would adding the `-AsUTC` provide the functionality you're thinking of? 

For example, the tests would be: 
```
Get-Date -Date 1577836800 -FromUnixTime | Should -Be (Get-Date -Date 637134336000000000 -AsUTC)
Get-Date -Date 0 -FromUnixTime | Should -Be (Get-Date -Date 621355968000000000 -AsUTC)
```
Please let me know if this is what you're thinking!

Reply:
I'd expect AsUTC is right thing here.

Reply:
Makes sense. Updated!